Source/Shaders/Builtin/Functions/cascadeDistance.glsl

```diff
@@ -1,4 +1,7 @@
+
+uniform vec4 u_shadowMapCascadeDistances;
```

Comment:
Ah, I think I have run into the same thing before and opted not to put a uniform with a function like this.  Pass them in.  You could use a `struct` if you trust GLSL compilers...could be OK.